readthedocs/projects/urls/private.py

```diff
@@ -250,17 +242,15 @@
 
 urlpatterns += domain_urls
 
-# We are allowing users to enable the new addons only from the new dashboard
-if settings.RTD_EXT_THEME_ENABLED:
-    addons_urls = [
-        re_path(
-            r"^(?P<project_slug>[-\w]+)/addons/edit/$$",
-            AddonsConfigUpdate.as_view(),
-            name="projects_addons",
-        ),
-    ]
+addons_urls = [
+    re_path(
+        r"^(?P<project_slug>[-\w]+)/addons/edit/$$",
```

Comment:
This was probably working just by chance?

```suggestion
        r"^(?P<project_slug>[-\w]+)/addons/edit/$",
```